app/views/categories/index.html.erb

```diff
@@ -0,0 +1,41 @@
+<% provide(:title, "Categories") %>
+<br>
+<div class="container">
+    <div class="row">
+        <div class="col s12">
+            <div class="card">
+                <div class="card-action red lighten-2 white-text">
+                    <h6>Categories</h6>
+                </div>
+                <%= will_paginate %>
+                <div class=row>
+                    <% @categories.each do |category| %>
+                    <div class="col s12 m6">
+                        <div class="card-content">
+                            <div class="something">
+                                <div class="card blue-grey darken-0">
+                                    <div class="card-content white-text">
+                                        <span class="card-title">
+                                            <h6><%= category.title %> </h6>
+                                        </span>
+                                        <p> <b>[ <%= category.words.count %> words ] </b> <%= category.description %>
+                                        </p>
+                                    </div>
+                                    <div class="card-action">
+                                        <%= form_for([category, @lesson]) do |f| %>
+                                            <%= f.hidden_field :user_id, :value => current_user.id %>
```

Comment:
@johnedelmapa
No need to pass `user` info, since `current_user` method can be accessed in controller scope as well.
This will prevent user from controlling user id information in client side